spec/models/manageiq/providers/redhat/infra_manager/provision/state_machine_spec_helper.rb

```diff
@@ -0,0 +1,65 @@
+module ManageIQ::Providers::Redhat::InfraManager::Provision::StateMachineSpecHelper
+  include MiqProvision::StateMachineSpecHelper
+
+  def test_create_destination
+    call_method
+  end
+
+  def test_determine_placement
+    call_method
+  end
+
+  def test_start_clone_task
+    expect(task).to receive(:start_clone)
+
+    call_method
+  end
+
+  def test_poll_clone_complete
+    @test_poll_clone_complete_setup ||= begin
+      expect(task).to receive(:clone_complete?).and_return(false, false, true)
+      expect(task).to receive(:requeue_phase).twice { requeue_phase }
```

Comment:
I'd be concerned as we add usage of more let's beyond `task` from the specs.  It might get hard to follow where these methods are defined.